_posts/current/blog/2020-05-07-linaro-engineering-highlights-april-2020-new-version-2.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: 'Linaro Engineering Highlights: April 2020'
+date: '2020-05-07 04:47:27'
+image: /assets/images/content/code.jpg
+tags:
+  - Linaro
+  - Engineering
+  - Highlights
+  - News
+  - April
+category: Blog
+author: jon.burcham@linaro.org
+---
+Welcome to the April 2020 edition of the Linaro Engineering Highlights. This is a roundup of all of the latest news and developments from last month including:- 
+
+* Linaro’s Contributions to the Linux v5.6 Kernel LTP (Linux Test Project) 
+* PSA Level 1 Certification Showcase
+* Linaro Consumer Group (LCG) News
+* OTA article - Industrial Internet Consortium Journal of Innovation
+* Protein Folding on Arm Devices - Helping with COVID-19 Research by Sahaj Sarup
+
+### Linaro’s Contributions to the Linux v5.6 Kernel
+
+###### **Mark Orvek, VP Engineering**
+
+{% include image.html path="/assets/images/content/core-eng.jpg" class="medium-inline left" alt="core-eng.jpg" %}
+
+As reported by LWN, the latest stable Linux Kernel (version 5.6) was released on March 29th. I wanted to highlight five notable Linaro related statistics from the article; you can read the full LWN article at [Some 5.6 kernel development statistics](lwn.net/Articles/816162/.):
```

Comment:
Broken link as the protocol is not included resulting in a local relative link. Please replace with:
[Some 5.6 kernel development statistics](https://lwn.net/Articles/816162/)